src/install/browserFetcher.ts

```diff
@@ -84,6 +86,7 @@ function getDownloadUrl(browserName: BrowserName, platform: BrowserPlatform): st
       ['mac10.13', undefined],
       ['mac10.14', '%s/builds/webkit/%s/minibrowser-mac-10.14.zip'],
       ['mac10.15', '%s/builds/webkit/%s/minibrowser-mac-10.15.zip'],
+      ['mac10.16', '%s/builds/webkit/%s/minibrowser-mac-10.15.zip'],
```

Comment:
I believe this requires a dedicated build similar to 10.14. We shouldn't provide a link for now unless it's working.

Reply:
tested on my big sur dev machine :)

Reply:
it could change when the actual big sur release is out.